examples-experimental/potree/src/potree-traverser.ts

```diff
@@ -0,0 +1,37 @@
+import {POTreeNode} from "@loaders.gl/potree";
+
+export class PotreeTraverser {
```

Comment:
If we must have a traverser class, can we generalize it a little bit: `PointTileSourceTraverser`?